src/Modules/SimplCommerce.Module.Catalog/Views/Product/ProductOverview.cshtml

```diff
@@ -0,0 +1,233 @@
+                    }
+                    else
+                    {
+                        <h3>@Model.CalculatedProductPrice.Price.ToString("C")</h3>
+                        if (Model.CalculatedProductPrice.PercentOfSaving > 0)
+                        {
+                            <p class="product-price-regular">@Model.CalculatedProductPrice.OldPrice.Value.ToString("C")</p>
+                            <p class="product-price-saving">You save (@(Model.CalculatedProductPrice.PercentOfSaving)%)</p>
+                        }
+                    }
+                </div>
+                if (Model.StockTrackingIsEnabled && Model.StockQuantity <= 0)
+                {
+                    <div class="out-of-stock">
+                        <span class="label label-danger">@Localizer["Out of stock"]</span>
+                    </div>
+                }
+                <div class="add-to-cart">
+                    <form class="inline">
+                        <input type="hidden" name="productId" value="@Model.Id" />
+                        <div class="product-quantity">
+                            <button type="button" class="quantity-button" name="subtract" value="-">-</button>
+                            <input type="text" class="quantity-field" readonly="readonly" name="qty" value="1">
+                            <button type="button" class="quantity-button" name="add" value="+">+</button>
+                        </div>
+                        <button type="button" disabled="@(!Model.IsAllowToOrder || (Model.StockTrackingIsEnabled && Model.StockQuantity <= 0))" class="btn btn-lg btn-add-cart">@Localizer["Add to cart"]</button>
+                        <div>
+                            <ul class="list-inline add-to-lists">
+                                <li class="wishlist-in">
+                                    <i class="fa fa-heart"></i>
+                                    @if (SignInManager.IsSignedIn(User))
+                                    {
+                                        <a href="#" class="add-to-wishlist">@Localizer["Add to Wishlist"]</a>
+                                    }
+                                    else
+                                    {
+                                        <a asp-controller="Account" asp-action="Login" asp-route-returnurl="@($"{Context.Request.Path}{Context.Request.QueryString}")">@Localizer["Add to Wishlist"]</a>
+                                    }
+                                </li>
+                                <li class="compare-in">
+                                    <i class="fa fa-exchange"></i>
+                                    <a href="#" class="add-to-comparison">Add to Compare</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </form>
+                </div>
+            }
+        </div>
+    </div>
+    @if (!string.IsNullOrWhiteSpace(Model.Description))
```

Comment:
I think the product overview should not contain Description, and Specification, these information can be very long

Reply:
sure I removed them